src/AccordionItem/AccordionItem.wrapper.js

```diff
@@ -32,20 +32,22 @@ class AccordionItemWrapper extends Component<AccordionItemWrapperProps> {
 
     static defaultProps = defaultProps;
 
+    renderItem = ({ accordionStore, itemStore }) => {
```

Comment:
Pretty much equivalent, except that by making `renderItem` a class instance instead of an anonymous function then the descendants will not be unmounted-remounted on every re-render, which (in this case) prevents a race-conditions-induced render loop.